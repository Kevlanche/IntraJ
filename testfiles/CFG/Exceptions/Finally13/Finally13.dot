digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"; "ClassDecl[92]"; "ClassDecl[99]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"; "Modifiers[93]"; "Opt[95]"; "List[97]"; "List[98]"; "Modifiers[100]"; "Opt[102]"; "List[104]"; "List[105]"}
{ rank = same; "List[7]"; "MethodDecl[12]"; "List[94]"; "TypeAccess[96]"; "List[101]"; "TypeAccess[103]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[18]"; "Opt[20]"; "Exit[21]"}
{ rank = same; "List[15]"; "TypeAccess[19]"; "Block[21]"}
{ rank = same; "List[22]"}
{ rank = same; "TryStmt[23]"; "VarDeclStmt[84]"}
{ rank = same; "Block[24]"; "List[56]"; "Opt[72]"; "UncheckedExceptions[83]"; "Modifiers[85]"; "PrimitiveTypeAccess[87]"; "List[88]"}
{ rank = same; "List[25]"; "BasicCatch[57]"; "Block[73]"; "List[86]"; "VariableDeclarator[89]"}
{ rank = same; "VarDeclStmt[26]"; "ThrowStmt[34]"; "ParameterDeclaration[58]"; "Block[62]"; "List[74]"; "List[90]"; "Opt[91]"}
{ rank = same; "Modifiers[27]"; "PrimitiveTypeAccess[29]"; "List[30]"; "ClassInstanceExpr[36]"; "UncheckedExceptions[35]"; "List[55]"; "Modifiers[59]"; "TypeAccess[61]"; "List[63]"; "VarDeclStmt[75]"}
{ rank = same; "List[28]"; "VariableDeclarator[31]"; "TypeAccess[37]"; "List[38]"; "Opt[39]"; "List[40]"; "List[41]"; "NTAFinallyBlock[43]"; "List[60]"; "VarDeclStmt[64]"; "Modifiers[76]"; "PrimitiveTypeAccess[78]"; "List[79]"}
{ rank = same; "List[32]"; "Opt[33]"; "List[44]"; "Modifiers[65]"; "PrimitiveTypeAccess[67]"; "List[68]"; "List[77]"; "VariableDeclarator[80]"}
{ rank = same; "Block[45]"; "List[66]"; "VariableDeclarator[69]"; "List[81]"; "Opt[82]"}
{ rank = same; "List[46]"; "List[70]"; "Opt[71]"}
{ rank = same; "VarDeclStmt[47]"}
{ rank = same; "Modifiers[48]"; "PrimitiveTypeAccess[50]"; "List[51]"}
{ rank = same; "List[49]"; "VariableDeclarator[52]"}
{ rank = same; "List[53]"; "Opt[54]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Finally13 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < m10 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[18]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[19]" [label="TypeAccess\n < TestException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[20]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[21]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[22]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TryStmt[23]" [label="TryStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[24]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[25]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[26]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[27]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[28]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[29]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[30]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[31]" [label="VariableDeclarator\n < int a >", fillcolor=white   style=filled  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[33]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ThrowStmt[34]" [label="ThrowStmt\n ", fillcolor=white   style=filled  ]
"ClassInstanceExpr[36]" [label="ClassInstanceExpr\n <TestException>", fillcolor=white   style=filled  ]
"TypeAccess[37]" [label="TypeAccess\n < TestException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[38]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[39]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[40]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[41]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"NTAFinallyBlock[43]" [label="NTAFinallyBlock\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[44]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[45]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[46]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[47]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[48]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[49]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[50]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[51]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[52]" [label="VariableDeclarator\n < int c >", fillcolor=white   style=filled  ]
"List[53]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[54]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[35]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"List[55]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[56]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BasicCatch[57]" [label="BasicCatch\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[58]" [label="ParameterDeclaration\n < e >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[59]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[60]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[61]" [label="TypeAccess\n < TestSubException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[62]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[63]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[64]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[65]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[66]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[67]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[68]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[69]" [label="VariableDeclarator\n < int b >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[70]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[71]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[72]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[73]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[74]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[75]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[76]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[77]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[78]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[79]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[80]" [label="VariableDeclarator\n < int c >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[81]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[82]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[83]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[84]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[85]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[86]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[87]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[88]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[89]" [label="VariableDeclarator\n < int d >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[90]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[91]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[21]" [label="Exit\n ", fillcolor=white   style=filled  ]
"ClassDecl[92]" [label="ClassDecl\n < TestException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[93]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[94]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[95]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[96]" [label="TypeAccess\n < Exception >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[97]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[98]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[99]" [label="ClassDecl\n < TestSubException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[100]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[101]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[102]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[103]" [label="TypeAccess\n < TestException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[104]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[105]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"List[18]" -> "TypeAccess[19]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[18]" [style=dashed, color=gray]
"Modifiers[27]" -> "List[28]" [style=dashed, color=gray]
"VarDeclStmt[26]" -> "Modifiers[27]" [style=dashed, color=gray]
"VarDeclStmt[26]" -> "PrimitiveTypeAccess[29]" [style=dashed, color=gray]
"VariableDeclarator[31]" -> "List[32]" [style=dashed, color=gray]
"VariableDeclarator[31]" -> "Opt[33]" [style=dashed, color=gray]
"List[30]" -> "VariableDeclarator[31]" [style=dashed, color=gray]
"VarDeclStmt[26]" -> "List[30]" [style=dashed, color=gray]
"List[25]" -> "VarDeclStmt[26]" [style=dashed, color=gray]
"ClassInstanceExpr[36]" -> "TypeAccess[37]" [style=dashed, color=gray]
"ClassInstanceExpr[36]" -> "List[38]" [style=dashed, color=gray]
"ClassInstanceExpr[36]" -> "Opt[39]" [style=dashed, color=gray]
"ClassInstanceExpr[36]" -> "List[40]" [style=dashed, color=gray]
"ClassInstanceExpr[36]" -> "List[41]" [style=dashed, color=gray]
"ThrowStmt[34]" -> "ClassInstanceExpr[36]" [style=dashed, color=gray]
"Modifiers[48]" -> "List[49]" [style=dashed, color=gray]
"VarDeclStmt[47]" -> "Modifiers[48]" [style=dashed, color=gray]
"VarDeclStmt[47]" -> "PrimitiveTypeAccess[50]" [style=dashed, color=gray]
"VariableDeclarator[52]" -> "List[53]" [style=dashed, color=gray]
"VariableDeclarator[52]" -> "Opt[54]" [style=dashed, color=gray]
"List[51]" -> "VariableDeclarator[52]" [style=dashed, color=gray]
"VarDeclStmt[47]" -> "List[51]" [style=dashed, color=gray]
"List[46]" -> "VarDeclStmt[47]" [style=dashed, color=gray]
"Block[45]" -> "List[46]" [style=dashed, color=gray]
"List[44]" -> "Block[45]" [style=dashed, color=gray]
"NTAFinallyBlock[43]" -> "List[44]" [style=dashed, color=gray]
"ThrowStmt[34]" -> "NTAFinallyBlock[43]" [style=dashed, color=gray]
"ThrowStmt[34]" -> "UncheckedExceptions[35]" [style=dashed, color=gray]
"ThrowStmt[34]" -> "List[55]" [style=dashed, color=gray]
"List[25]" -> "ThrowStmt[34]" [style=dashed, color=gray]
"Block[24]" -> "List[25]" [style=dashed, color=gray]
"TryStmt[23]" -> "Block[24]" [style=dashed, color=gray]
"Modifiers[59]" -> "List[60]" [style=dashed, color=gray]
"ParameterDeclaration[58]" -> "Modifiers[59]" [style=dashed, color=gray]
"ParameterDeclaration[58]" -> "TypeAccess[61]" [style=dashed, color=gray]
"BasicCatch[57]" -> "ParameterDeclaration[58]" [style=dashed, color=gray]
"Modifiers[65]" -> "List[66]" [style=dashed, color=gray]
"VarDeclStmt[64]" -> "Modifiers[65]" [style=dashed, color=gray]
"VarDeclStmt[64]" -> "PrimitiveTypeAccess[67]" [style=dashed, color=gray]
"VariableDeclarator[69]" -> "List[70]" [style=dashed, color=gray]
"VariableDeclarator[69]" -> "Opt[71]" [style=dashed, color=gray]
"List[68]" -> "VariableDeclarator[69]" [style=dashed, color=gray]
"VarDeclStmt[64]" -> "List[68]" [style=dashed, color=gray]
"List[63]" -> "VarDeclStmt[64]" [style=dashed, color=gray]
"Block[62]" -> "List[63]" [style=dashed, color=gray]
"BasicCatch[57]" -> "Block[62]" [style=dashed, color=gray]
"List[56]" -> "BasicCatch[57]" [style=dashed, color=gray]
"TryStmt[23]" -> "List[56]" [style=dashed, color=gray]
"Modifiers[76]" -> "List[77]" [style=dashed, color=gray]
"VarDeclStmt[75]" -> "Modifiers[76]" [style=dashed, color=gray]
"VarDeclStmt[75]" -> "PrimitiveTypeAccess[78]" [style=dashed, color=gray]
"VariableDeclarator[80]" -> "List[81]" [style=dashed, color=gray]
"VariableDeclarator[80]" -> "Opt[82]" [style=dashed, color=gray]
"List[79]" -> "VariableDeclarator[80]" [style=dashed, color=gray]
"VarDeclStmt[75]" -> "List[79]" [style=dashed, color=gray]
"List[74]" -> "VarDeclStmt[75]" [style=dashed, color=gray]
"Block[73]" -> "List[74]" [style=dashed, color=gray]
"Opt[72]" -> "Block[73]" [style=dashed, color=gray]
"TryStmt[23]" -> "Opt[72]" [style=dashed, color=gray]
"TryStmt[23]" -> "UncheckedExceptions[83]" [style=dashed, color=gray]
"List[22]" -> "TryStmt[23]" [style=dashed, color=gray]
"Modifiers[85]" -> "List[86]" [style=dashed, color=gray]
"VarDeclStmt[84]" -> "Modifiers[85]" [style=dashed, color=gray]
"VarDeclStmt[84]" -> "PrimitiveTypeAccess[87]" [style=dashed, color=gray]
"VariableDeclarator[89]" -> "List[90]" [style=dashed, color=gray]
"VariableDeclarator[89]" -> "Opt[91]" [style=dashed, color=gray]
"List[88]" -> "VariableDeclarator[89]" [style=dashed, color=gray]
"VarDeclStmt[84]" -> "List[88]" [style=dashed, color=gray]
"List[22]" -> "VarDeclStmt[84]" [style=dashed, color=gray]
"Block[21]" -> "List[22]" [style=dashed, color=gray]
"Opt[20]" -> "Block[21]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[20]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[21]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"Modifiers[93]" -> "List[94]" [style=dashed, color=gray]
"ClassDecl[92]" -> "Modifiers[93]" [style=dashed, color=gray]
"Opt[95]" -> "TypeAccess[96]" [style=dashed, color=gray]
"ClassDecl[92]" -> "Opt[95]" [style=dashed, color=gray]
"ClassDecl[92]" -> "List[97]" [style=dashed, color=gray]
"ClassDecl[92]" -> "List[98]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[92]" [style=dashed, color=gray]
"Modifiers[100]" -> "List[101]" [style=dashed, color=gray]
"ClassDecl[99]" -> "Modifiers[100]" [style=dashed, color=gray]
"Opt[102]" -> "TypeAccess[103]" [style=dashed, color=gray]
"ClassDecl[99]" -> "Opt[102]" [style=dashed, color=gray]
"ClassDecl[99]" -> "List[104]" [style=dashed, color=gray]
"ClassDecl[99]" -> "List[105]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[99]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[21]" -> "VariableDeclarator[52]" [color=red, constraint=false]
"VariableDeclarator[52]" -> "Exit[21]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[52]" -> "UncheckedExceptions[35]" [color=red, constraint=false]
"UncheckedExceptions[35]" -> "VariableDeclarator[52]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[35]" -> "ClassInstanceExpr[36]" [color=red, constraint=false]
"ClassInstanceExpr[36]" -> "UncheckedExceptions[35]" [color=blue, constraint=false, xlabel=" "] 
"ClassInstanceExpr[36]" -> "ThrowStmt[34]" [color=red, constraint=false]
"ThrowStmt[34]" -> "ClassInstanceExpr[36]" [color=blue, constraint=false, xlabel=" "] 
"ThrowStmt[34]" -> "VariableDeclarator[31]" [color=red, constraint=false]
"VariableDeclarator[31]" -> "ThrowStmt[34]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[31]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "VariableDeclarator[31]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"ClassDecl[5]" -> "ClassDecl[92]" -> "ClassDecl[99]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[18]" -> "Opt[20]" -> "Exit[21]" [style=invis]
"TryStmt[23]" -> "VarDeclStmt[84]" [style=invis]
"Block[24]" -> "List[56]" -> "Opt[72]" [style=invis]
"VarDeclStmt[26]" -> "ThrowStmt[34]" [style=invis]
"Modifiers[27]" -> "PrimitiveTypeAccess[29]" -> "List[30]" [style=invis]
"List[32]" -> "Opt[33]" [style=invis]
"TypeAccess[37]" -> "List[38]" -> "Opt[39]" [style=invis]
"Modifiers[48]" -> "PrimitiveTypeAccess[50]" -> "List[51]" [style=invis]
"List[53]" -> "Opt[54]" [style=invis]
"ParameterDeclaration[58]" -> "Block[62]" [style=invis]
"Modifiers[59]" -> "TypeAccess[61]" [style=invis]
"Modifiers[65]" -> "PrimitiveTypeAccess[67]" -> "List[68]" [style=invis]
"List[70]" -> "Opt[71]" [style=invis]
"Modifiers[76]" -> "PrimitiveTypeAccess[78]" -> "List[79]" [style=invis]
"List[81]" -> "Opt[82]" [style=invis]
"Modifiers[85]" -> "PrimitiveTypeAccess[87]" -> "List[88]" [style=invis]
"List[90]" -> "Opt[91]" [style=invis]
"Modifiers[93]" -> "Opt[95]" -> "List[97]" -> "List[98]" [style=invis]
"Modifiers[100]" -> "Opt[102]" -> "List[104]" -> "List[105]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
106", fillcolor=1]
Edges [label="Total Nr. Edges
117", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
7", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
6", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}