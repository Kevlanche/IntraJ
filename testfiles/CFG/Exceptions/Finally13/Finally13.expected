NodeName: Entry[13] Successors: [VariableDeclarator[31]]
NodeName: VariableDeclarator[31] Successors: [ThrowStmt[34]]
NodeName: ThrowStmt[34] Successors: [ClassInstanceExpr[36]]
NodeName: ClassInstanceExpr[36] Successors: [UncheckedExceptions[35]]
NodeName: UncheckedExceptions[35] Successors: [VariableDeclarator[52]]
NodeName: VariableDeclarator[52] Successors: [Exit[21]]
NodeName: Exit[21] Successors: []

NodeName: Exit[21] Predecessor: [VariableDeclarator[52]]
NodeName: VariableDeclarator[52] Predecessor: [UncheckedExceptions[35]]
NodeName: UncheckedExceptions[35] Predecessor: [ClassInstanceExpr[36]]
NodeName: ClassInstanceExpr[36] Predecessor: [ThrowStmt[34]]
NodeName: ThrowStmt[34] Predecessor: [VariableDeclarator[31]]
NodeName: VariableDeclarator[31] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
